src/applications/check-in/tests/e2e/pages/ValidateVeteran.js

```diff
@@ -33,6 +35,14 @@ class ValidateVeteran {
     this.getLast4Input().type(last4);
   }
 
+  clearLastName() {
```

Comment:
The input fields preserve the values after a failed attempt, so I added clear functions to reset the values of last4 and lastName before typing. Cypress does have a .clear() function for this, but it was causing some issues here, so I went with `invoke` instead